setup.cfg

```diff
@@ -1,21 +1,13 @@
-[bdist_wheel]
-universal = 1
-
-[flake8]
-exclude = docs
-
-[aliases]
-test = pytest
-
-[tool:pytest]
-addopts = -v -x --doctest-modules --ignore=setup.py --cov=skhep_testdata --pep8
-pep8maxlinelength = 120
-norecursedirs = src dev
-
 [metadata]
-name = scikit-hep-testdata
+name = scikit_hep_testdata
+description = Example HEP files for testing and demonstrating
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/scikit-hep/scikit-hep-testdata
 author = Ben Krikler
 author_email = b.krikler@cern.ch
+license = BSD-3-Clause
```

Comment:
It probably makes sense to add at this stage
```
maintainer = The Scikit-HEP admins
maintainer_email = scikit-hep-admins@googlegroups.com
```
as we have in other packages (e.g. `Particle`) since Ben is not maintaining nor working on this package for 2 years or so, I would guess. But we keep maintaining the package :-).

Reply:
I'm impressed it's persisted this far even - for future PRs feel free to change any parts of this as clearly you're now doing a much a larger amount of maintenance than I am!

Reply:
We can add an author / authors if the package becomes much more or totally different but for now it makes total sense to acknowledge your writing ;-). My 2 cents.

Reply:
I'm happy with the current state - author is the original author, that's why there's a maintainer slot too. :) (though we can add authors if there's a need)